README.md

```diff
@@ -26,62 +26,48 @@ Please return to this page regularly to stay updated with the progress of develo
 
 ## For Developers
 
-It is recommended to work on N-Trans in a virtual environment.
+N-Trans is split into three phases.
 
-To set up a virtual environment:
-- MAC OS:
-  `python3 -m venv env`
-
-To activate a virtual environment:
-- MAC OS:
-  `source env/bin/activate`
+1) In `ntrans_dataprep.py`, sentences from the BNC are processed and, split into N-grams, and written
+to .csv files in chunks of 300K sentences.
 
+The script generates a total of 236.937.946 N-grams.
 
+2) In `ntrans_compile.py`, the frequency of each N-gram is counted with `collections.Counter`. The 50K
```

Comment:
Perhaps it should be renamed to `ntrans_combine`, since it combines the separate files?